pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTest.java

```diff
@@ -206,8 +207,8 @@ public void testSegmentStatusChecker()
     int numTables = 5;
     ControllerMetrics controllerMetrics = _controllerStarter.getControllerMetrics();
     TestUtils.waitForCondition(aVoid -> {
-      if (controllerMetrics
-          .getValueOfGlobalGauge(ControllerGauge.PERIODIC_TASK_NUM_TABLES_PROCESSED, "SegmentStatusChecker")
+      if (MetricValueUtils.getGaugeValue(controllerMetrics,
+          ControllerGauge.PERIODIC_TASK_NUM_TABLES_PROCESSED.getGaugeName() + ".SegmentStatusChecker")
```

Comment:
Similar comment here, it is best to leave this as a task name argument, at least then you can enforce naming that sticks to some convention there and not let anybody who adds new stuff come up with a new way to concatenate names across the code